packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -409,6 +409,16 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
         this.context.setCurrentCallId(this.call.id);
       }
 
+      // if we have a call state we need to dispose of the unparented views
```

Comment:
nit: update this comment to describe the scenario of why it exists to help prevent it being deleted in future
```suggestion
// If the call connects we need to clean up any previous unparentedViews
```

Reply:
From this -- should we also clean up if it goes into the disconnected or disconnecting states too?

Reply:
Actually should we clean up every state except `connecting`?